modules/Cookie Consent/classes/CookieConsent.php

```diff
@@ -2,7 +2,7 @@
 /**
  * CookieConsent module main class
  *
- * @package Modules\Cookie Consent
+ * @package Modules\CookieConsent
```

Comment:
Noticed these weren't showing on the PHPDoc site, I think because of the space here